interface/src/instruction.rs

```diff
@@ -731,6 +731,24 @@ pub enum TokenInstruction<'a> {
     ScaledUiAmountExtension,
     /// Instruction prefix for instructions to the pausable extension
     PausableExtension,
+    // 45
+    /// Transfer lamports from a native SOL account to a destination account.
+    ///
+    /// This is useful to unwrap lamports from a wrapped SOL account.
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    ///   0. `[writable]` The source account.
+    ///   1. `[writable]` The destination account.
+    ///   2. `[signer]` The source account's owner/delegate.
+    ///
+    UnwrapLamports {
+        /// The amount of lamports to transfer. When an amount is
+        /// not specified, the entire balance of the source account will be
+        /// transferred.
+        #[cfg_attr(feature = "serde", serde(with = "coption_fromstr"))]
```

Comment:
Maybe this is ok, since generally we should do this more for u64 values, but we don't normally serialize amounts as strings.